flang/lib/Decimal/decimal-to-binary.cpp

```diff
@@ -16,6 +16,10 @@
 #include <ctype.h>
 #include <utility>
 
+// Some environments, viz. glibc 2.17, allow the macro HUGE
+// to leak out of <math.h>.
```

Comment:
Please mention in this comment that FreeBSD also allows the macro HUGE to leak out of <math.h>.

Reply:
I've added a second commit to update all the comments.